nengo/utils/least_squares_solvers.py

```diff
@@ -69,6 +69,10 @@ def __call__(self, A, y, sigma, rng=None):
         info = {'rmses': rmses(A, x, y)}
         return x, info
 
+    def __eq__(self, other):
```

Comment:
This is a FrozenObject, so I don't think this should be necessary. Is this just a holdover from before it was a FrozenObject?